### PR TITLE
Fix: Fix workspace error

### PR DESCRIPTION
**Pull request by bbaa77770**

### CODE REVIEW

**Review by xiekun1992 — APPROVED** [review]

File: src/app/ui/widgets/Marlin/Laser.jsx [hunk 1]
```diff
@@ -117,47 +115,45 @@ class Laser extends PureComponent {
...
+                            name="Reset"
+                            type={['static']}
+                            className="border-default-black-5 margin-left-4 border-radius-8"
+                            onClick={actions.onSaveLaserPower}
+                            borderRadius={8}
+                            disabled={isPrinting}
+                        />
                     </div>
-                )}
-                {!isWifiPrinting && (
```
Comment (jane-rose) [inline src/app/ui/widgets/Marlin/Laser.jsx:159]: only for high power?

File: src/app/ui/widgets/Marlin/Laser.jsx [hunk 2]
```diff
@@ -44,37 +47,36 @@ class Laser extends PureComponent {
...
-                this.props.executeGcode('M3 P1 S2.55');
-                // this.props.executeGcode(`M3 P${this.state.laserPower} S${this.state.laserPower * 255 / 100}`);
-                // if (this.state.laserPower > 1) {
-                //     this.props.executeGcode('G4 P500');
-                //     this.props.executeGcode('M3 P1 S2.55');
-                // }
+                if (this.props.toolHead === LEVEL_TWO_POWER_LASER_FOR_SM2) {
+                    this.props.executeGcode('M3 P1 S2.55');
+                } else {
+                    this.props.executeGcode(`M3 P${this.state.laserPower} S${this.state.laserPower * 255 / 100}`);
```
Comment (jane-rose) [inline src/app/ui/widgets/Marlin/Laser.jsx:70]: M3 P?

File: src/app/ui/widgets/WorkspaceVisualizer/Visualizer.jsx [hunk 3]
```diff
@@ -265,6 +267,8 @@ class Visualizer extends PureComponent {
             const { connectionType } = this.props;
             if (connectionType === CONNECTION_TYPE_SERIAL) {
                 const { workflowState } = this.state;
+                this.props.executeGcode('G0 X0 Y0 F1000');
+                this.props.executeGcode(`G0 Z${this.props.materialThickness ?? 0} F1000`);
```
Comment (jane-rose) [inline src/app/ui/widgets/WorkspaceVisualizer/Visualizer.jsx:271]: only execute this gcode for high power

File: src/app/ui/widgets/Marlin/Laser.jsx (hunk 2 above)
Comment (bbaa77770) [reply src/app/ui/widgets/Marlin/Laser.jsx:70]: Normally open in laser 1.6w

**Review by jane-rose — APPROVED** [review]

**Pull request merged**